### PR TITLE
[Fix] Fix the script gather_models.

### PR DESCRIPTION
**Pull request by Czm369**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Fix the script gather_models.

## Modification

Fix the script gather_models.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMCls.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

**Comment by xuwanzhuang** [comment]
z

File: .dev_scripts/gather_models.py [hunk 1]
```diff
@@ -212,11 +183,15 @@ def convert_model_info_to_pwc(model_infos):
 def parse_args():
     parser = argparse.ArgumentParser(description='Gather benchmarked models')
     parser.add_argument(
-        'root',
+        '--root',
```
Comment (hhaAndroid) [inline .dev_scripts/gather_models.py:186]:
```suggestion
        'root',
```

File: .dev_scripts/gather_models.py [hunk 2]
```diff
@@ -212,11 +183,15 @@ def convert_model_info_to_pwc(model_infos):
 def parse_args():
```
Comment (hhaAndroid) [inline .dev_scripts/gather_models.py:183]: Have you tested epoch-based and iteration-based methods?

Comment (Czm369) [reply .dev_scripts/gather_models.py:183]: Yep

**Review by hhaAndroid — APPROVED** [review]

**Pull request merged**